visualization/app/codeCharta/state/store/files/files.reducer.spec.ts

```diff
@@ -0,0 +1,115 @@
+import {
+	addFile,
+	defaultFiles,
+	FilesAction,
+	resetFiles,
+	setDelta,
+	setDeltaByNames,
+	setFiles,
+	setMultiple,
+	setMultipleByNames,
+	setSingle,
+	setSingleByName
+} from "./files.actions"
+import { Files } from "../../../model/files"
+import { TEST_DELTA_MAP_A, TEST_DELTA_MAP_B } from "../../../util/dataMocks"
+import files from "./files.reducer"
+
+describe("files", () => {
+	let state = new Files()
+
+	beforeEach(() => {
+		files(state, resetFiles())
+		files(state, addFile(TEST_DELTA_MAP_A))
+		files(state, addFile(TEST_DELTA_MAP_B))
+	})
+
+	describe("Default State", () => {
+		it("should initialize the default state", () => {
+			const result = files(undefined, {} as FilesAction)
+
+			expect(result).toEqual(defaultFiles)
+		})
+	})
+
+	describe("Action: SET_FILES", () => {
+		it("should set new files", () => {
+			const newFiles = new Files()
+			newFiles.addFile(TEST_DELTA_MAP_A)
+
+			const result = files(state, setFiles(newFiles))
+
+			expect(result).toEqual(newFiles)
+		})
+
+		it("should set default files", () => {
+			const result = files(state, setFiles())
+
+			expect(result).toEqual(new Files())
```

Comment:
Do you wanna use the imported `defaultFiles` here like above?
```suggestion
			expect(result).toEqual(defaultFiles)
```